Core/StitchCoreSDK/Sources/StitchCoreSDK/StitchError.swift

```diff
@@ -109,6 +109,7 @@ public enum StitchServiceErrorCode: String, Codable {
     notCallable = "FunctionNotCallable",
     userAlreadyConfirmed = "UserAlreadyConfirmed",
     userNotFound = "UserNotFound",
+    userNotLoggedIn = "UserNotLoggedIn",
```

Comment:
[minor] see my other comments about errors. this should be moved to `StitchClientErrorCode`, and you should also add `userNotFound` to `StitchClientErrorCode`